brianlong/www.validators.app
# 178913633-deduplicate-vote-accounts

#### What's this PR do?
Only one validator_history is saved for a validator in one batch

#### How should this be manually tested?
run gather_rpc script and see if there's only one validator_history saved for each validator

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/178913633

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
